server/api/routes-auth.js

```diff
@@ -117,17 +290,47 @@ if (typeof consts.MOCK_AUTHENTICATION_ENDPOINT === 'string') {
      */
     ctx.body = {
       access_token:
-        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.' +
-        'eyJhdWQiOlsiYmFzIl0sInVzZXJfbmFtZSI6ImFkbWluIiwic' +
-        '2NvcGUiOlsicmVhZCJdLCJleHAiOjk5OTk5OTk5OTk5OTksIn' +
-        'VzZXJJZCI6IjQwMjg4YjdlNWJjZDc3MzMwMTViY2Q3ZmQ3MjI' +
-        'wMDAxIiwiYXV0aG9yaXRpZXMiOlsiYWRtaW4iXSwianRpIjoi' +
-        'NzJlYzNjNDMtMDMwYS00MWVkLWFiYjItYjdhMjY5NTA2OTIzI' +
-        'iwiY2xpZW50X2lkIjoiYmFzLWNsaWVudCJ9.' +
-        'uwywziNetHyfSdiqcJt6XUGy4V_WYHR4K6l7OP2VB9I'
+      'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.' +
+      'eyJhdWQiOlsiYmFzIl0sInVzZXJfbmFtZSI6ImFkbWluIiwic' +
+      '2NvcGUiOlsicmVhZCJdLCJleHAiOjk5OTk5OTk5OTk5OTksIn' +
+      'VzZXJJZCI6IjQwMjg4YjdlNWJjZDc3MzMwMTViY2Q3ZmQ3MjI' +
+      'wMDAxIiwiYXV0aG9yaXRpZXMiOlsiYWRtaW4iXSwianRpIjoi' +
+      'NzJlYzNjNDMtMDMwYS00MWVkLWFiYjItYjdhMjY5NTA2OTIzI' +
+      'iwiY2xpZW50X2lkIjoiYmFzLWNsaWVudCJ9.' +
+      'uwywziNetHyfSdiqcJt6XUGy4V_WYHR4K6l7OP2VB9I'
     }
   })
-}
+
+  router.get(ENDPOINT_BACKEND_VALIDATE, async (ctx) => {
+    console.log(`Mocking a response for ${ctx.url}`)
+    let fakeIsValid = false
+    // Just mimicking we only accept as a valid session the hardcoded JWT token
+    // from ENDPOINT_BACKEND_AUTH above.
+    const tokenBeginsWith = /^Token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9\./.test(ctx.querystring)
+    if (tokenBeginsWith) {
+      fakeIsValid = true
+    }
+    // When API returns strings, we will handle at validate
+    const Status = fakeIsValid ? 'valid' : 'invalid'
+    const validated = {
+      Status
+    }
+    if (fakeIsValid) {
+      validated.UserInfo = {
+        UserName: 'admin',
+        DisplayName: 'Haaw D. Minh',
```

Comment:
This is a joke I’ve created to myself and left there.
If this is inappropriate, please tell me :)

Reply:
Talking about easter egg in the code.
In my previous PR, I've added a city called Lyster. I used an utf based emojii of a palm tree for chinese.